GenderPayGap.WebUI/ExternalServices/DataRepositories/IDataTransaction.cs

```diff
@@ -1,21 +0,0 @@
-﻿using System;
-using System.Threading.Tasks;
-
-namespace GenderPayGap.Core.Interfaces
-{
-    public interface IDataTransaction
-    {
-
-        /// <summary>
-        ///     Creates a transaction for aggregating data update operations
-        /// </summary>
-        /// <param name="transactionFunc"></param>
-        Task BeginTransactionAsync(Func<Task> transactionFunc);
-
-        void CommitTransaction();
-
-        void RollbackTransaction();
-
-    }

```

Comment:
This is now unused (was only being used by `UpdateFromCompaniesHouseService` which has no need to be transactional)